plugins/community/roadrunner-server-php-grpc/v4.2.0/buf.plugin.yaml

```diff
@@ -0,0 +1,11 @@
+version: v1
+name: buf.build/community/roadrunner-server-php-grpc
+plugin_version: v4.2.0
```

Comment:
Are we going to use `v4.3.0` and update to `go install` ?

Reply:
Updated.